modules/repository-azure/licenses/jna-NOTICE.txt

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Nit: the other notice files with no contents are empty, I suggest to remove the newline here.

Reply:
We enforce in our build that all instances of the `NOTICE` file for each library are the same. This is a copy of the already-extant `x-pack/plugin/sql/sql-cli/licenses/jna-NOTICE.txt`.